package.json

```diff
@@ -56,6 +56,8 @@
         "displayName": "Organizations: Permissions required to call services apart from mod-organizations-storage",
         "visible": false,
         "subPermissions": [
+          "acquisitions-units.memberships.collection.get",
```

Comment:
in case permissions are changed it requires corresponding migration script for the back-end. At least related ticket for now

Reply:
created story [MODORG-10](https://issues.folio.org/browse/MODORG-10)